README.md

```diff
@@ -216,6 +216,41 @@ The benefits of using the plain version of SDK are:
 - The ability to scope CMA client instance to a specific `spaceId`, `environmentId`, and `organizationId` when initializing the client.
   - You can pass a concrete values to `defaults` and omit specifying these params in actual CMA methods calls.
 
+### App Framework
+
+Starting [`@contentful/app-sdk@4`](https://github.com/contentful/app-sdk) you can use this client to issue requests
```

Comment:
I think this is more explicit. Alternative: `@contentful/app-sdk` v4

```suggestion
Starting [`@contentful/app-sdk` Version 4](https://github.com/contentful/app-sdk) you can use this client to issue requests
```

Reply:
Not changing this to stick with existing convention (see a bit below in the Readme)